{{cookiecutter.project_slug}}/clients/web/react/package.json

```diff
@@ -6,7 +6,7 @@
     {% if cookiecutter.use_graphql == 'n' -%}
     "@tanstack/react-query": "^4.24.2",
     "axios": "^1.2.6",
-    "@thinknimble/tn-models-fp": "2.5.3",
+    "@thinknimble/tn-models-fp": "^2.11.4",
```

Comment:
@paribaker looks like we should also be adding `zod` as a dependency since we use it on these new auth service